uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-assignment.js

```diff
@@ -79,15 +82,32 @@ define([
                 $scope.dpOpened[key] = true;
             };
 
-            $scope.refreshContacts = function (input, type) {
+            /**
+             * Fetches and stores a list of contacts based on the given
+             * query string
+             *
+             * @param {string} input - The search query the user typed
```

Comment:
To be more consistent, I'd put the description in a new line, just like the following param descriptions.